config/config.go

```diff
@@ -36,6 +36,14 @@ var validIssueTrackers = []IssueTracker{
 
 var windowsAbsolutePathPattern = regexp.MustCompile("^[A-Z]{1}:")
 
+var originPatterns = map[IssueTracker]*regexp.Regexp{
+	IssueTrackerJira:    regexp.MustCompile(`[a-zA-Z0-9\-]+\.[a-zA-Z]+$`),
+	IssueTrackerGithub:  regexp.MustCompile(`^(https://)?(www\.)?github\.com/[a-zA-Z0-9\-_]+/[a-zA-Z0-9\-_]+`),
+	IssueTrackerGitlab:  regexp.MustCompile(`^(https://)?(www\.)?gitlab\.com/[a-zA-Z0-9\-_]+/[a-zA-Z0-9\-_]+`),
```

Comment:
Some companies have a private instance of gitlab setup. E.g. `https://mygitlab.org.com`.

`todocheck` has support for private gitlab instances but this validation excludes them.

Example - #40 

This pattern should (similarly to jira & redmine) match any (sub)domain.

Reply:
fixed in 56054bf and 4b08267 with test case in defa509